src/Components/AddContacts/AddContact.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import { Button, View, StyleSheet } from 'react-native';
+import 'react-native-get-random-values';
+import { v4 as uuidv4 } from 'uuid';
+import { useDispatch } from 'react-redux';
+import { useNavigation } from '@react-navigation/native';
+
+import FiledForm from './FiledForm';
+
+import colors from '../../Constants/colors';
+import { CONTACTS } from '../../Constants/navigationRouts';
+import { ADD_CONTACT } from '../../Actions/contacts';
+
+const AddContact = () => {
+  const navigation = useNavigation();
+  const dispatch = useDispatch();
+
+  const [name, setName] = useState('');
+  const [surname, setSurname] = useState('');
+  const [phoneNumber, setPhoneNumber] = useState('');
+  const id = uuidv4();
+
+  const onPress = () => {
+    if (name && phoneNumber) {
+      dispatch(
+        ADD_CONTACT({
+          id,
+          name,
+          surname,
+          avatar_image: '',
+          phone_number: phoneNumber,
+          favorites: false,
+        }),
```

Comment:
remove Id generation from here.
do it inside reducer.

see example [https://redux-toolkit.js.org/api/createAction#using-prepare-callbacks-to-customize-action-contents](url),

use nanoId or your integrated uuid there